content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,219 @@
+To have the report grouped appropriately, click the :icon:`fa-caret-down` :guilabel:`(down caret)`
+icon, to the right of the search bar, and click :guilabel:`Salesperson` in the :guilabel:`Group
+By` section. All results are now grouped by the salesperson assigned to each lead.
+
+Filter for quality leads
+------------------------
+
+The following additional conditions are provided as an example of a *good*, but *not comprehensive*,
+set of rules for finding quality leads. These filters should be applied on top of the
+:ref:`crm/track_leads/essential-filters` in the order specified to achieve a heavily-detailed
+filter.
+
+- **Referred-by:** Filter for referrals, such as by appointment or sales member.
+
+- **Source:** Filter for specific source UTMs, such as Facebook or LinkedIn.
+
+- **Notes:** Filter for internal notes.
+
+- **Tags:** Filter for categorical tags.
+
+- **Email:** Filter for specific email domains, such as gmail.com or yahoo.com.
+
+- **Salesperson:** Filter for leads associated with certain sales members.
+
+These conditions can be added, removed, or modified to best fit the desired information in
+the report.
+
+.. seealso::
+  `Additional rules for quality leads <https://www.odoo.com/documentation/17.0/applications/sales/
+  crm/track_leads/quality_leads_report.html#add-rules-for-quality-leads>`_
+
+.. tip::
+
+  Once the rules for the filter are set, click the purple :guilabel:`Confirm` button at the
+  bottom of the pop-up menu to save the custom filter and close the pop-up menu.
+
+  .. image:: lead_distribution_report/confirm-filter.png
+    :align: center
+    :alt: Click the confirm button to save the filter.
+
+
+  The :guilabel:`Pipeline Analysis` dashboard is now displayed again with each filter rule in the
+  search bar. Click the :icon:`fa-caret-down` :guilabel:`(down caret)` icon, to the
+  right of the search bar, to reveal a drop-down menu of search and filter options.
+
+  Scroll to the bottom and click the :guilabel:`Save current search` button. Next, type a name for
+  the filter in the text box below. Check the :guilabel:`Shared` checkbox to have the filter shared
+  with any user with access to the pipeline. Finally, click the purple :guilabel:`Save` button below
+  to save the filter.
+
+  The filter will now appear with the name it was given under the :guilabel:`Favorites` section of
+  the drop-down menu and can be re-applied by clicking on it.
+
+View lead distribution reports
+==============================
+
+From the :guilabel:`Pipeline Analysis` dashboard, apply the filter created above in the search bar
+if it does not appear in the search already. Next, click the :icon:`fa-caret-down` :guilabel:`(down
+caret)` icon to expand the list and view leads.
+
+.. image:: lead_distribution_report/list-view.png
+   :align: center
+   :alt: List view will display all leads in a list format organized by the filter.
+
```

Comment:
Don't think this section is necessary?
After we click **Confirm** we get the output we want, this is enough right?

Also we can't use that screenshot (again do not use screenshots taken from internal DB). 

I *do* think showing an outputted view is useful tho with a saved filter, something like this (took from CRM tour):
![image](https://github.com/user-attachments/assets/133dfdf8-17f8-4a5c-b4eb-f09cd47d322d)